guides/common/modules/con_overview-of-recommended-disaster-recovery-plans.adoc

```diff
@@ -54,3 +54,16 @@ You must ensure that content synchronization and content view creation are synch
 Content drift occurs when available content deviates from the intended state defined by a content view.
 If you fail to prevent content drift, expect inconsistency in the content that is available to hosts.
 endif::[]
+
+ifeval::["{context}" == "planning"]
```

Comment:
I'd strongly recommend using the `:planning:` attribute instead of context. We want to get rid of contexts where they aren't needed for reuse.
```suggestion
ifdef::planning[]
```
Please, define it in the `master.adoc`.

Reply:
This is a very nice idea which simplifies things a lot! Thanks Lena.